src/main/java/seedu/address/logic/parser/EditAssignmentCommandParser.java

```diff
@@ -24,25 +25,43 @@ public EditAssignmentCommand parse(String args) throws ParseException {
         requireNonNull(args);
         ArgumentMultimap argumentMultimap =
                 ArgumentTokenizer.tokenize(args, PREFIX_STUDENT_INDEX, PREFIX_ASSIGNMENT_INDEX,
-                        PREFIX_ASSIGNMENT_SCORE);
-        if (!arePrefixesPresent(argumentMultimap, PREFIX_STUDENT_INDEX, PREFIX_ASSIGNMENT_INDEX,
-                PREFIX_ASSIGNMENT_SCORE)) {
+                        PREFIX_ASSIGNMENT_NAME, PREFIX_ASSIGNMENT_MAX_SCORE);
+        if (!arePrefixesPresent(argumentMultimap, PREFIX_STUDENT_INDEX, PREFIX_ASSIGNMENT_INDEX)) {
```

Comment:
I assume that they are able to either change the assignmentName or maxScore or both at the same time, which is why there are only checks for student index and assignment index?

Reply:
Great observation! This if condition here is intended to check missing prefixes, and only the two indices are must-have prefixes. As you said, the assignmentName and maxScore are optional to be changed.